pyanaconda/users.py

```diff
@@ -27,8 +27,11 @@
 import pwquality
 from pyanaconda.iutil import strip_accents
 from pyanaconda.constants import PASSWORD_MIN_LEN
+
 from pyanaconda.errors import errorHandler, PasswordCryptError, ERROR_RAISE
 
+from re import search as regsearch
```

Comment:
``import re`` and then using ``re.search`` would be better readable. This way when I hit the ``regsearch`` identifier in the code it looks to me like some custom thing.